rbac/management/tenant_service/v2.py

```diff
@@ -294,13 +413,42 @@ def _bootstrap_tenant(self, tenant: Tenant) -> BootstrappedTenant:
         return BootstrappedTenant(tenant, mapping, default_workspace=default_workspace, root_workspace=root_workspace)
 
     def _bootstrap_default_access(
-        self, tenant: Tenant, mapping: TenantMapping, default_workspace: Workspace
+        self, tenant: Tenant, mapping: TenantMapping, default_workspace_uuid: str, custom_groups=None
     ) -> List[Relationship]:
         """
         Bootstrap default access for a tenant's users and admins.
 
         Creates role bindings between the tenant's default workspace, default groups, and system policies.
         """
+
+        def _create_default_relation_tupples(
```

Comment:
```suggestion
        def _create_default_relation_tuples(
```

This one could maybe also be moved to a top-level method so the _bootstrap_default_access is a little less busy? It's defined as an inner method but doesn't look like utilizes variables in that method's scope for what it's worth (unless I'm missing something).

Reply:
Moved to top level